client/deposits/list/index.tsx

```diff
@@ -217,7 +217,6 @@ export const DepositsList = (): JSX.Element => {
 
 	const endpointExport = async ( language: string ) => {
 		// We destructure page and path to get the right params.
-		// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Whoops! Looks like removing this linter suppression triggered a few warnings. Are these really unused variables and can be removed? Or they are used somewhere and we should restore the suppression?

Reply:
I attempted to eliminate the params variable, but doing so resulted in other errors. It seems the suppression was originally included for a reason, so I re-added it.